mods/cnc/maps/gdi07/rules.yaml

```diff
@@ -2,7 +2,7 @@ World:
 	LuaScript:
 		Scripts: campaign.lua, utils.lua, gdi07.lua
 	MissionData:
-		Briefing: Previous mission objective not complete.\nAirfield was to be targeted. \n\nNew objective:  Build up a base and Destroy remaining Nod structures and units.\n\nReinforcements will be provided.
+		Briefing: Previous mission objective not complete.\nAirfield remains intact. \n\nNew objective:  Build up a base and Destroy remaining Nod structures and units.\n\nReinforcements will be provided.
```

Comment:
This changes the meaning (that the airfield specifically was the target, vs incidentally remained), keep previous wording.